packages/framework/src/Resources/views/Admin/Content/Pricing/Groups/pricingGroupSettings.html.twig

```diff
@@ -1,14 +1,16 @@
 {{ form_start(form) }}
-    <div class="wrap-border form-group">
-        <div class="form-group__title">
-            {{ 'Settings'|trans }}
+    <div class="wrap-divider">
+        <div class="form-group">
+            <div class="form-group__title">
+                {{ 'Settings'|trans }}
+            </div>
+            {{ form_row(form.defaultPricingGroup, { 'label': 'Default pricing group'|trans }) }}
         </div>
-        {{ form_row(form.defaultPricingGroup, { 'label': 'Default pricing group'|trans }) }}
-    </div>
 
-    {% embed '@ShopsysFramework/Admin/Inline/FixedBar/fixedBar.html.twig' %}
-        {% block fixed_bar_content %}
-            {{ form_widget(form.save, { label: 'Save changes'|trans}) }}
-        {% endblock %}
-    {% endembed %}
-{{ form_end(form) }}
\ No newline at end of file
+        {% embed '@ShopsysFramework/Admin/Inline/FixedBar/fixedBar.html.twig' %}
+            {% block fixed_bar_content %}
+                {{ form_widget(form.save, { label: 'Save changes'|trans}) }}
+            {% endblock %}
+        {% endembed %}
+    </div>
```

Comment:
is it important to have the div after embed ? Wouldn't be better to move it before `{% embed` block ?